CONTRIBUTING.md

```diff
@@ -73,7 +73,7 @@ export POSTGRES_PASS="db_pass"
 export POSTGRES_DB="db_name"
 # Postgres instance with IAM authentication enabled
 # (can be same as POSTGRES_CONNECTION_NAME)
-export POSTGRES_IAM_CONNECTION_NAME="<PROJECT>:<REGION>:<INSTANCE>"
+export POSTGRES_CONNECTION_NAME="<PROJECT>:<REGION>:<INSTANCE>"
```

Comment:
we should just remove this line as we already have an export line for this variable above on line 70...

We should copy the comment `# Postgres instance with IAM authentication enabled` to above line 70.

Reply:
Also should do the same for `MYSQL_IAM_CONNECTION_NAME`